Scripts/git-cp.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I'd still add a header comment explaining why this exists, since it doesn't seem useful without the context of splitting.